app/Metadata/GitHubFunctions.php

```diff
@@ -69,10 +59,6 @@ private function trim($commit_id)
 	 */
 	public function get_current_branch()
 	{
-		if (App::runningUnitTests()) {
```

Comment:
Are we sure that omitting that condition has no negative impact?

[`GitHubFunctions::count_behind(bool)`](https://github.com/LycheeOrg/Lychee/blob/bc918a394a995a939b6e5651746895c464370a2d/app/Metadata/GitHubFunctions.php#L127) checks for the master branch. `count_behind` is called in `is_up_to_date` and I have not checked further where this method is used. Anyway, I can't oversee right now (and haven't check) which other methods further down the call graph rely on `get_current_branch` to fake the master branch during tests.

Reply:
Nah I don't think this impact in the count_behind. I would expect the tests to break otherwise. :)

Reply:
OK, then let's be brave. If any bug occurs, we will fix it then.